A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js

```diff
@@ -71,14 +70,15 @@ class A320_Neo_EICAS extends Airliners.BaseEICAS {
         this.updateAnnunciations();
         this.updateScreenState();
         
-        var engineOn = Simplane.getEngineActive(0) || Simplane.getEngineActive(1);
-        var externalPower = SimVar.GetSimVarValue("EXTERNAL POWER ON", "bool");
-        var apuOn = SimVar.GetSimVarValue("L:APU_GEN_ONLINE", "bool");
-        let apuStatusChanged = (apuOn != this.APULastValue);
-        let externalPowerChanged = (externalPower != this.externalPowerLastValue);
-
-        var isACPowerAvailable = engineOn || apuOn || externalPower;
+        const engineOn = Simplane.getEngineActive(0) || Simplane.getEngineActive(1);
+        const externalPower = SimVar.GetSimVarValue("EXTERNAL POWER ON", "bool");
```

Comment:
setting this to const is breaking the game because later down in this function there is `var externalPower = ...` which ends up throwing an error.

Reply:
Weird, looks like this change wasn't staged somehow, my bad. Nice that you fixed it already.